packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -73,6 +75,28 @@ interface Label {
   name: string;
 }
 
+/**
+ * Reduces the list of PRs to search by half if there are more than 100 entries at a time
+ * @returns an array of Datastore PRs
+ */
+handler.maybeReducePRList = function maybeReducePRList(prs: DatastorePR[]) {
+  if (prs.length >= MAX_ENTRIES) {
+    if (Date.now() % 2 === 0) {
+      prs = prs.filter(x => new Date(x.created!).getTime() % 2 === 0);
+      logger.info(
+        `Too many entries in Datastore table; examining first ${prs.length}`
+      );
+    } else {
```

Comment:
No need for the `else`, the modulus step will grab the appropriate sized chunk of work, time is essentially just being treated as a random value for us.